daemon/mgr/container.go

```diff
@@ -386,6 +386,32 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		return nil, err
 	}
 
+	// check disk quota
+	// TODO: regexp check and one quota for rootfs and all of volumes.
+	if meta.Config.DiskQuota == nil {
+		if v, ok := meta.Config.Labels["DiskQuota"]; ok {
+			meta.Config.DiskQuota = make(map[string]string)
+
+			parts := strings.Split(v, ";")
+			for _, quota := range parts {
```

Comment:
the following 10 lines of code is repeated from the client side. I am wondering if we could DRY to encapsulate two duplicated into one single function.